templates/blog/article.html

```diff
@@ -4,14 +4,14 @@
 {% block title %}{{ article.title }}, by {{ article.author|capitalize }}{% endblock %}
 {% block description %}{{ article.content|truncate(200) }} by {{ article.author|capitalize }}.{% endblock %}
 
-{% block extra_og %}
-{% if article.image %}
-<meta property=og:image content="{% if article.image[:4] == 'http' %}{{ article.image }}{% else %}{{ request.url_root[:-1] }}{{ article.image }}{% endif %}">
-{% endif %}
-<meta property=og:article:published_time content="{{ article.creation_date }}">
-<meta property=og:article:author content="{{ article.author|capitalize }}">
-<meta property=og:description content="Article « {{ article|striptags|truncate(200) }} », by {{ article.author|capitalize }} on {{ article.creation_date|datetime }}">
-{% endblock %}
+{% block extra_og -%}
+  {% if article.image -%}
```

Comment:
why you finish with `-%}`? new way to write ? should we use everywhere for `block` and `if` ?

Reply:
`-%}` is a writing I hadn't understood so far. It removes the next newline.
I guess we could use it everywhere we create a newline for the html